pleskdistup/actions/systemd.py

```diff
@@ -176,6 +175,33 @@ def _revert_action(self) -> action.ActionResult:
         return action.ActionResult()
 
 
+# We need to handle the mailman service separately because it is not configured and is stopped by default
+# If we try to start an unconfigured service, it will fail, causing issues during the revert and finish stages
+class HandleMailmanService(action.ActiveAction):
+    mailman_service: str
+
+    def __init__(self) -> None:
+        self.name = "handle mailman service"
+        self.mailman_service = "mailman.service"
+
+    def _is_required(self) -> bool:
```

Comment:
I suppose we are not calling prepare, post, revert if is_REquired is False, right?

Reply:
Yep, you right